code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -178,6 +178,15 @@
 	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "chemical_laughter", /datum/mood_event/chemical_laughter)
 	..()
 
+/datum/reagent/consumable/laughter/expose_mob(mob/living/M, method=TOUCH, reac_volume)
+	var/mob/living/carbon/human/reactor = M
+	if(istype(reactor))
+		var/datum/component/mood/mood = reactor.GetComponent(/datum/component/mood)
+		if (mood.get_event("slipped"))
+			SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "laughter", /datum/mood_event/funny_prank)
+			SEND_SIGNAL(M, COMSIG_CLEAR_MOOD_EVENT, "slipped")
+			reactor.AdjustKnockdown(-20)
```

Comment:
```suggestion
			reactor.AdjustKnockdown(-2 SECONDS)
```
might wanna also consider buffing this up to -4 SECONDS, since this is an exposure effect that can only happen once per slipping